src/puya/awst/to_code_visitor.py

```diff
@@ -502,6 +502,11 @@ def visit_tuple_expression(self, expr: nodes.TupleExpression) -> str:
             items = ", ".join([item.accept(self) for item in expr.items])
         return f"({items})"
 
+    @typing.override
+    def visit_named_tuple_expression(self, expr: nodes.NamedTupleExpression) -> str:
+        items = ", ".join([f"{name}={value.accept(self)}" for name, value in expr.values.items()])
```

Comment:
The list comprehension is technically redundant here, although I can see it's come from copying the behaviour for NewStruct also.